content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -72,13 +72,17 @@ you already have at-rest encryption enabled. However, that check does not tell y
 a previous migration to encrypted storage has succeeded. If you are not sure, see
 [ensure all relevant data are encrypted](#ensure-all-secrets-are-encrypted).
 
-{{< caution >}}
-**IMPORTANT:** For high-availability configurations (with two or more control plane nodes), the
-encryption configuration file must be the same! Otherwise, the `kube-apiserver` component cannot
-decrypt data stored in the etcd.
-{{< /caution >}}
+## Plan an encryption-at-rest configuration {#understanding-the-encryption-at-rest-configuration}
+
+{{< note >}}
+For cluster configurations with two or more control plane nodes, the encryption configuration
+should be identical across each control plane node.
```

Comment:
This is repeated as a caution block later in the file (under the heading ”Reconfigure other control plane hosts”).

I think the advice bears repeating. Get this wrong and you can really end up regretting it.